Tests/Container/ContainerTest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+class ContainerTest extends TestCase
+{
+}
```

Comment:
This Container test doesn't have any tests.

Please add at least one test to do `Container` class test.

Reply:
Yes you're right

Reply:
`use Zest\Container\Container` I added this statement first.  Because we did not have any test cases there, the usage of that class did not occur in it. So an automatic code check was complaining about it. Then later I removed that statement. I can add a test case to it. Can anyone suggest a test here? :)

Reply:
You may looks to others frameworks test in case of `Laravel` ....

Reply:
@Lablnet  and @peter279k, I have implemented one test for container and also modified two functions in container. It will be great if you look at it. :)